c/include/cuvs/neighbors/cagra.h

```diff
@@ -422,6 +498,25 @@ cuvsError_t cuvsCagraIndexGetSize(cuvsCagraIndex_t index, int64_t* size);
  */
 cuvsError_t cuvsCagraIndexGetGraphDegree(cuvsCagraIndex_t index, int64_t* graph_degree);
 
+/**
+ * @brief Check if the CAGRA index is stored on disk
+ *
+ * @param[in] index CAGRA index
+ * @param[out] on_disk return true if index is on disk, false otherwise
+ * @return cuvsError_t
+ */
+cuvsError_t cuvsCagraIndexIsOnDisk(cuvsCagraIndex_t index, bool* on_disk);
+
+/**
+ * @brief Get the file directory where the CAGRA index is stored (if on disk)
```

Comment:
I think these docs should expand on why a user would call these functions. It's unclear to me from the docs how the algorithm requires (or doesn't) the use of these functions).